modules/installing-rhv-requirements.adoc

```diff
@@ -2,7 +2,7 @@
 //
 // * installing/installing_rhv/installing-rhv-custom.adoc
 // * installing/installing_rhv/installing-rhv-default.adoc
-// * installing/installing_rhv/installing-rhv-user-infra.adoc
+// * installing/installing_rhv/installing-rhv-restricted-network.adoc
```

Comment:
The UPI assembly is also using this module, so I think the restricted network reference here should be on a new line, and not replace the UPI one.